### DISCUSSION TITLE
What is the correct docker compose image to use?

### DISCUSSION THREAD

**Discussion by prairietree** [blakeblackshear/frigate#13801] [Ask A Question] [answered]
I have been using `image: ghcr.io/blakeblackshear/frigate:stable` in my docker compose file. That is the image tag recommended in the documentation (https://docs.frigate.video/frigate/installation#docker). However, it seems to be stuck at version 0.13.2. Is that expected or am I doing something wrong? I would like to update to version 14 but I do not see any tags for that other than each releases tag. Thanks for you help.

**Comment by NickM-27** [comment, marked as answer]
you are using the correct tag, you just need to use docker compose update

**Comment by prairietree** [comment]
I think you are correct. Every time I look up how to update with docker compose I find a different answer. I was using:

```
docker-compose down
docker-compose build --pull
docker-compose up -d
```

But after looking some more, I think the correct way might be:

```
docker-compose down
docker-compose up -d --build --pull always
```

**Reply by NickM-27** [reply]
You can also just do 

```
docker compose update
docker compose up -d
```

**Reply by prairietree** [reply]
Thanks, after doing the docker compose update correctly, I am now at version 0.14.1-f4f3cfa.
